ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -75,6 +75,15 @@ type HeadGroupSpec struct {
 	// EnableIngress indicates whether operator should create ingress object for head service or not.
 	// +optional
 	EnableIngress *bool `json:"enableIngress,omitempty"`
+	// Resources specifies the resource quantities for the head group.
+	// These values override the resources passed to `rayStartParams` for the group.
+	// +optional
+	Resources corev1.ResourceList `json:"resources,omitempty"`
+	// Labels specifies the Ray node labels for the head group.
+	// These labels will also be added to the Pods of this head group and override the `--labels`
```

Comment:
This should mention that labels are ignored if already specifeid in `rayStartParams`?